packages/botonic-nlp/src/utils/array-utils.ts

```diff
@@ -1,3 +1,7 @@
 export function unique<T>(array: T[]): T[] {
   return Array.from(new Set(array))
 }
+
+export function randomSort<T>(array: T[]): T[] {
```

Comment:
rename to shuffle
apply this algorithm https://stackoverflow.com/a/12646864/145289
here explains why the solution in the PR is not optimal https://stackoverflow.com/a/18650169/145289